doc/services/identity/v3/tokens.rst

```diff
@@ -43,3 +43,37 @@ Revoke token
 
 .. sample:: identity/v3/tokens/revoke_token.php
 .. refdoc:: OpenStack/Identity/v3/Service.html#method_revokeToken
+
+Cache authentication token
+--------------------------
+
+Use case
+~~~~~~~~
+
+Before one can start calling any API, a very first step that this SDK will do is to authenticate with Identity service
```

Comment:
Before the SDK performs an API call, it will first authenticate to the OpenStack Identity service using the provided credentials.